src/appshell/qml/dockwindow/DockSeparator.qml

```diff
@@ -39,8 +39,8 @@ Rectangle {
         anchors.fill: parent
         anchors.margins: -5 //! NOTE: extra space for user convenience
 
-        cursorShape: separatorCpp ? (separatorCpp.isVertical ? Qt.SizeVerCursor : Qt.SizeHorCursor)
-                                   : Qt.SizeHorCursor
+        cursorShape: (separatorCpp && separatorCpp.showResizeCursor) ? (separatorCpp.isVertical ? Qt.SizeVerCursor : Qt.SizeHorCursor)
+                                   : Qt.SizeArrowCursor
```

Comment:
Is `Qt.SizeArrowCursor` correct? Didn't you mean `Qt.ArrowCursor`? (https://doc.qt.io/qt-6/qt.html#CursorShape-enum)

Reply:
Interesting, it must just default to ArrowCursor if you use an undefined type! But yes, obviously a typo.